### DISCUSSION TITLE
Sphinx-doc for experimental.jet?

### DISCUSSION THREAD

**Discussion by pnkraemer** [jax-ml/jax#9536] [Ideas] [answered]
Hi everyone,

Big fan of `jet` here! I came across #3730 recently, but since it might be a little out of scope for the issue, I thought I'd start a mew discussion here, instead of turning a seemingly resolved issue into a discussion. :) If _ideas_ is the wrong category, please feel free to move the discussion.

I noticed that some experimental APIs are documented (`pjit`, `maps`, ...), while others aren't (`jet`, `ode`,...). **Is there a specific rule regarding which experimental APIs are documented and which ones are not?**

Because if it is a matter of someone writing the docs, I'd be very happy to contribute a sphinx-doc for `jax.experimental.jet` with some basic explanations and examples! 

I made some notes about the API for myself recently (based on https://openreview.net/pdf?id=SkxEF3FNPH, and my own user experience), and could turn this into a pull request fairly quickly I guess. A rough draft is attached below, but I would obviously include only what you want me to include! :) 

Thoughts?

![Screenshot from 2022-02-11 17-04-05](https://user-images.githubusercontent.com/45484438/153625887-28ee30b3-9d46-43aa-9930-ab101f33d8d5.png)

![Screenshot from 2022-02-11 17-06-20](https://user-images.githubusercontent.com/45484438/153626202-5331e764-d712-4f2c-86b3-16cdad0d7957.png)

**Comment by hawkinsp** [comment, marked as answer]
I'm sure we'd welcome a PR that includes `jet` in the docs. We don't mind documenting experimental things, `experimental` is simply a statement about the expected level of support and quality.

**Comment by jakevdp** [comment]
To answer your bolded question: no, I don't think we have any rule regarding which experimental packages are included in the HTML docs. But looking  at [jet.py](https://github.com/google/jax/blob/main/jax/experimental/jet.py), it's clear there hasn't yet been any effort put into module or function-level docstrings, and this is probably why it's not yet been included in the documentation build. As Peter says, a PR improving that situation would be very welcome.

**Comment by pnkraemer** [comment]
Thanks for the quick feedback @hawkinsp and @jakevdp. I'll look into it then :+1:

**Reply by pnkraemer** [reply]
See #9546.
